Sources/Clappr_iOS/Classes/Plugin/Core/Drawer/BottomDrawerPlugin.swift

```diff
@@ -57,15 +57,21 @@ open class BottomDrawerPlugin: DrawerPlugin {
     }
 
     private func moveUp() {
+        enableUserInteractionOnFirstSubview(true)
         view.setVerticalPoint(to: size.height, duration: ClapprAnimationDuration.mediaControlShow)
     }
 
     private func moveDown(with duration: TimeInterval = ClapprAnimationDuration.mediaControlHide) {
+        enableUserInteractionOnFirstSubview(false)
         view.setVerticalPoint(to: hiddenHeight, duration: duration)
     }
 
-    private func addGesture(_ gesture: UIGestureRecognizer) {
-        gesture.cancelsTouchesInView = false
+    private func enableUserInteractionOnFirstSubview(_ enabled: Bool) {
+        self.view.subviews.first?.isUserInteractionEnabled = enabled
+    }
+
+    private func addGesture(_ gesture: UIGestureRecognizer, cancelsTouchesInView: Bool = false) {
```

Comment:
```suggestion
    private func addGesture(_ gesture: UIGestureRecognizer, cancelingTouchesInView: Bool = true) {
```

Considering that `cancelsTouchesInView` default value is true on UIGestureRecognizers, I think we should invert the default parameter to also be true. This would avoid confusion by making clear we're altering the default behavior on the call site.

Also, just changed the verb tense so it reads better. What are your thoughts on this?

Reply:
That makes sense to me! 
👍